### PR TITLE
Adding limits for AWS i3 platform for storageDeviceSets

### PR DESCRIPTION
**Pull request by vavuthu**
Signed-off-by: vavuthu <vavuthu@redhat.com>

### CODE REVIEW

**Comment by vavuthu** [comment]
@nehaberry Do we need to set limits for other platforms ( LSO ). from code it was set for all platforms previously.

File: ocs_ci/deployment/deployment.py [hunk 1]
```diff
@@ -599,6 +599,13 @@ def deploy_ocs_via_operator(self, image=None):
                 and not lso_type == constants.AWS_EBS
             ):
                 deviceset_data["count"] = 2
+            # setting resource limits for AWS i3
+            # https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.6/html-single/deploying_openshift_container_storage_using_amazon_web_services/index#creating-openshift-container-storage-cluster-on-amazon-ec2_local-storage
+            if ocs_version >= 4.5 and self.platform.lower() == constants.AWS_PLATFORM:
```
Comment (petr-balogh) [inline ocs_ci/deployment/deployment.py:604]: Can we add it only for:
worker_instance_type == 'i3en.2xlarge'
?
As we have also arbiter lso deployment on AWS and this should not be there.

Comment (vavuthu) [reply ocs_ci/deployment/deployment.py]: Done

**Review by clacroix12 — APPROVED** [review]

**Review by petr-balogh — APPROVED** [review]

**Comment by vavuthu** [comment]
updated with correct limits

```
name: ocs-deviceset-localblock
    placement: {}
    portable: false
    replica: 3
    resources:
      limits:
        cpu: 2
        memory: 5Gi
      requests:
        cpu: 1
        memory: 5Gi
```

**Pull request merged**